webhook_server/libs/handlers/check_run_handler.py

```
@@ -229,15 +233,15 @@ async def set_check_run_status(
 
         try:
             self.logger.debug(f"{self.log_prefix} Set check run status with {kwargs}")
-            await asyncio.to_thread(self.github_webhook.repository_by_github_app.create_check_run, **kwargs)
-            if conclusion in (SUCCESS_STR, IN_PROGRESS_STR):
-                self.logger.success(msg)  # type: ignore
-            return
-
+            await self.unified_api.create_check_run(self.github_webhook.repository_by_github_app, **kwargs)
         except Exception as ex:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)